edi_af_officer/messages/af_get_officer.py

```diff
@@ -55,7 +59,7 @@ def unpack(self):
                 if not office:
                     _logger.info("office number %s not in database, creating" % officer.get('officeCode'))
                     office = self.env['hr.department'].create({'name': officer.get('officeCode'), 'office_code': officer.get('officeCode'), 'note': _('Missing in AIS-F')})
-                if office and len(office) == 1:
+                if office and len(office) == 1: # hr.department should have a constraint added to it so length doesn't need to be checked
```

Comment:
Lots of commented out code, do we need to keep it?

Reply:
yes, it's unfinished code that hopefully may need to be finished in the future.